docs/compatibility/PackageValidation.md

```diff
@@ -0,0 +1,243 @@
+    {
+        // call unix specific stuff
+    }
+#else
+    public static void Open(string path)
+    {
+        // call windows specific stuff
+    }
+#endif
+```
+
+The resulting package structure looks like
+
+```xml
+lib/net6.0/A.dll 
+runtimes/unix/lib/net6.0/A.dll
+```
+
+`lib\net6.0\A.dll` will always be used at compile time regardless of the underlying operating system. `lib\net6.0\A.dll` will also be used at runtime for non-Unix systems, but `runtimes\unix\lib\net6.0\A.dll` will be used at runtime for Unix systems.
+
+When you try to pack this project, you get an error:
+
+![MultipleRuntimes](MultipleRuntimes.png)
+
+you quickly realize your mistake and adds `A.B.Open(string)` to the unix runtime as well.
+
+```c#
+#if Unix
+    public static void Open(string path, bool securityDescriptor)
+    {
+        // call unix specific stuff
+    }
+    
+    public static void Open(string path)
+    {
+        // throw not supported exception
+    }
+#else
+    public static void Open(string path)
+    {
+        // call windows specific stuff
+    }
+#endif
+```
+
+You try to pack the project again.
+
+![MultipleRuntimesSuccessful](MultipleRuntimesSuccessful.png)
+
+## Suprresing Compatibilty Errors
+
+The compatibility errors for intentional changes can be suppressed by adding the ```CompatibilitySuppressions.xml``` file to your project.
+You can generate this file automatically by passing ```/p:GenerateCompatibilitySuppressionFile=true``` while building the project from the commandline or setting ```<GenerateCompatibilitySuppressionFile>true</GenerateCompatibilitySuppressionFile>``` in your project.
+
+The suppression file will look like this
+```
+<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:A.B.DoStringManipulation(System.String)</Target>
+    <Left>lib/netstandard2.0/A.dll</Left>
+    <Right>lib/net6.0/A.dll</Right>
+  </Suppression>
+</Suppressions>
+```
+
+where 
+DiagnosticID =  The DiagnosticId representing the error to be suppressed. The detailed list could be found [here](DiagnosticIds.md).
+
+Target = The target of where to suppress the diagnostic ids<see cref="DiagnosticId"/>.
+
+Left = Left operand of an APICompat comparison.
+
+Right = Right operand of an APICompat comparison.
+
+isBaseline =  true <see langword="true"/> if the suppression is to be applied to a baseline validation. false <see langword="false"/> otherwise.
+
+## Extra features
+
+1) Strict mode:- By default, Package validation runs the api-compat in non-strict mode. Enabling strict mode will change some rules and some other rules will be executed when getting the differences. This is useful when you want both sides we are comparing to be strictly the same on their surface area and identity. This could be enabled for different validators by setting 
```

Comment:
```suggestion
1) Strict mode:- By default, Package validation runs api-compat in non-strict mode. Enabling strict mode will change some rules and some other rules will be executed when getting the differences. This is useful when you want both sides we are comparing to be strictly the same on their surface area and identity. This could be enabled for different validators by setting 
```